app/src/main/res/navigation/navigation_graph.xml

```diff
@@ -9,18 +9,57 @@
         android:id="@+id/welcomeFragment"
         android:name="org.fossasia.openevent.general.welcome.WelcomeFragment"
         android:label="WelcomeFragment"
-        tools:layout="@layout/fragment_welcome" />
+        tools:layout="@layout/fragment_welcome" >
+        <action
+            android:id="@+id/action_welcome_to_search"
+            app:destination="@id/searchLocationFragment"
+            app:popEnterAnim="@anim/slide_in_left"
+            app:popExitAnim="@anim/slide_out_right"
+            app:enterAnim="@anim/slide_in_right"
+            app:exitAnim="@anim/slide_out_left"/>
+    </fragment>
     <fragment
         android:id="@+id/eventsFragment"
         android:name="org.fossasia.openevent.general.event.EventsFragment"
         android:label="EventsFragment"
-        tools:layout="@layout/fragment_events" />
+        tools:layout="@layout/fragment_events" >
+        <action
+            android:id="@+id/action_events_to_searchLocation"
+            app:destination="@id/searchLocationFragment"
+            app:popEnterAnim="@anim/slide_in_left"
+            app:popExitAnim="@anim/slide_out_right"
+            app:enterAnim="@anim/slide_in_right"
+            app:exitAnim="@anim/slide_out_left"/>
+        <action
+            android:id="@+id/action_events_to_eventsDetail"
+            app:destination="@id/eventDetailsFragment"
+            app:popEnterAnim="@anim/fade_in"
+            app:popExitAnim="@anim/fade_out"
+            app:enterAnim="@anim/fade_in"
+            app:exitAnim="@anim/fade_out"/>
+        <action
+            android:id="@+id/action_events_to_searchResults"
+            app:destination="@id/searchResultsFragment"
+            app:popEnterAnim="@anim/slide_in_left"
+            app:popExitAnim="@anim/slide_out_right"
+            app:enterAnim="@anim/slide_in_right"
+            app:exitAnim="@anim/slide_out_left"/>
+
+    </fragment>
     <fragment
         android:id="@+id/similarEventsFragment"
         android:name="org.fossasia.openevent.general.event.topic.SimilarEventsFragment"
         android:label="SimilarEventsFragment"
         tools:layout="@layout/fragment_similar_events">
 
+        <action
+            android:id="@+id/action_similarEvents_to_eventDetails"
```

Comment:
fusion of snake case and camel case?

Reply:
@aggarwalpulkit596 Please use snake case only according to code style guideline.

Reply:
@nikit19 @liveHarshit will update them